electron/main.js

```diff
@@ -92,8 +91,25 @@ if (process.platform !== 'darwin') {
 ///////////////////////////////////////////////////////////////////////////////
 // IPC events
 ///////////////////////////////////////////////////////////////////////////////
+function getBaseUrl() {
+  let baseURL = argv.env || config.PRODUCTION;
+  if (!argv.env && config.DEVELOPMENT) {
+    let env = init.restore('env', 'internal');
+
+    if (env === 'cryptobox') baseURL = config.BENNY_URL;
+    if (env === 'dev') baseURL = config.DEV_URL;
+    if (env === 'edge') baseURL = config.EDGE_URL;
+    if (env === 'internal') baseURL = config.INTERNAL_URL;
+    if (env === 'localhost') baseURL = config.LOCALHOST_URL;
+    if (env === 'production') baseURL = config.PRODUCTION_URL;
```

Comment:
These strings here (i.e. `production`) are also reused in `developer.js`. Since they are used in two different places, we should have a constant for them, used in both places.